examples/userapi/05_conditionaldimension.ipynb

```diff
@@ -0,0 +1,729 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ConditionalDimension tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to demonstrate how users can exploit Devito's ConditionalDimension in order to express loop structures that contain conditionals. It is known that Devito can generate a great variety of loop kernels using its Domain Specific Language. This tutorial introduces the user in building conditional loop structures through a series of examples. We show not only the DSL API used but the generated code as well.\n",
```

Comment:
wrap ConditionalDimension between quotes , here and everywhere

Reply:
For the sake of brevity, I think this whole paragraph could and should be trimmed down to a single sentence:

```
This tutorial explains how to create equations that only get executed with the occurrence of given conditions.
```

Reply:
Fixed.